code/datums/mind.dm

```diff
@@ -590,8 +591,9 @@
 	if(SSticker.mode.config_tag == "devil")
 		text = uppertext(text)
 	text = "<i><b>[text]</b></i>: "
-	if(src in SSticker.mode.devils)
-		if(devilinfo && !devilinfo.ascendable)
+	var/datum/antagonist/devil/devilinfo = get_devil_datum(src)

```

Comment:
This won't work since you return `mind && mind.has_antag_datum`. Use only has_antag_datum for this